src/styles/avatar.scss

```diff
@@ -23,22 +23,22 @@ $block: #{$fd-namespace}-avatar;
   );
```

Comment:
Please remove `tabindex`

<img width="668" alt="Screenshot 2021-11-04 at 12 39 46" src="https://user-images.githubusercontent.com/20265336/140299946-fbcb1852-1616-405a-8004-95059d656e33.png">

Reply:
Unrelated to this PR